firestore-translate-text/functions/src/index.ts

```diff
@@ -172,6 +162,13 @@ const extractOutput = (snapshot: admin.firestore.DocumentSnapshot): any => {
   return snapshot.get(config.outputFieldName);
 };
 
+const extractLanguages = (
+  snapshot: admin.firestore.DocumentSnapshot
+): string[] => {
+  if (!config.languagesFieldName) return config.languages;
+  return snapshot.get(config.languagesFieldName) || config.languages;
```

Comment:
Wdyt of updating this to be a listener when the functions starts. Otherwise, we would incur a `read` for every translation made?

Reply:
sounds good

Reply:
Redacting suggestion as it is not applicable to this PR.